packages/elements-react/src/components/card/card-two-step.tsx

```diff
@@ -89,7 +89,7 @@ export function OryTwoStepCard() {
             {step === ProcessStep.ChooseAuthMethod && (
               <>
                 {flowType === FlowType.Login && (
-                  <BackButton href={config.project.login_ui_url} />
+                  <BackButton href={config.login_ui_url ?? '/login'} />
```

Comment:
I don't think we should make assumptions about the library setup here. If we do, this will cause unexpected behavior for customers.

Reply:
Hm ok, but it kinda sucks that I have to define all of these values when I want to use the SDK